runtime/src/main/java/org/corfudb/protocols/wireprotocol/CorfuMsgType.java

```diff
@@ -51,6 +51,7 @@
     TOKEN_RES(21, new TypeToken<CorfuPayloadMsg<TokenResponse>>(){}),
     BOOTSTRAP_SEQUENCER(22, new TypeToken<CorfuPayloadMsg<SequencerTailsRecoveryMsg>>(){}),
     SEQUENCER_TRIM_REQ(23, new TypeToken<CorfuPayloadMsg<Long>>() {}),
+    SEQUENCER_STATUS_REQ(24, TypeToken.of(CorfuMsg.class)),
```

Comment:
toIs this API really necessary? Shouldn't it be enough just to make a sequencer query and check if it fails?

Reply:
Hmm, I think so.